migtests/scripts/run-test.sh

```diff
@@ -168,6 +168,11 @@ main() {
 	expected_file="${TEST_DIR}/export_data_status-report.json"
```

Comment:
oh nice, didn't realise we already validate this!

Reply:
Yes! We validate the status reports but do not exhaustively do that for all tests.